src/app/(main)/layout.tsx

```diff
@@ -43,10 +45,17 @@ export default async function MainLayout({
         Skip to main content
       </a>
       <Header globalConfig={globalContent} />
+      {/* uncomment to have banner centered */}
+      {/* <div className="govuk-width-container"> */}
       <Banner globalConfig={globalContent} />
-      <main className="layout-wrap" id="main">
```

Comment:
The `layout-wrap` class styles still exist in `_layout.scss` - please remove them if no longer used. If the next PRs do this then ignore this 😄

Reply:
I'm saving that for the last PR - its all so interconnected rn